### PR TITLE
security/clamav: add custom signatures

### PR DESCRIPTION
**Pull request by mimugmail**
Related to #376

### CODE REVIEW

File: security/clamav/src/opnsense/mvc/app/models/OPNsense/ClamAV/Url.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            <list type="ArrayField">
+                <enabled type="BooleanField">
+                    <default>1</default>
+                    <Required>Y</Required>
+                </enabled>
+                <name type="TextField">
+                    <Required>Y</Required>
+                </name>
+                <link type="TextField">
+                    <Required>Y</Required>
```
Comment (fabianfrz) [inline security/clamav/src/opnsense/mvc/app/models/OPNsense/ClamAV/Url.xml]: should have a mask like starts with http at least

File: security/clamav/src/opnsense/mvc/app/views/OPNsense/ClamAV/general.volt [hunk 2]
```diff
@@ -34,6 +34,7 @@ POSSIBILITY OF SUCH DAMAGE.
 
 <ul class="nav nav-tabs" data-tabs="tabs" id="maintabs">
     <li class="active"><a data-toggle="tab" href="#general">{{ lang._('General') }}</a></li>
+    <li><a data-toggle="tab" href="#lists">{{ lang._('Lists') }}</a></li>
```
Comment (fabianfrz) [inline security/clamav/src/opnsense/mvc/app/views/OPNsense/ClamAV/general.volt]: I would call that external signatures

Comment (mimugmail) [reply security/clamav/src/opnsense/mvc/app/views/OPNsense/ClamAV/general.volt]: Would be a bit long for a naming a tab 🤔

File: security/clamav/src/opnsense/mvc/app/models/OPNsense/ClamAV/Url.xml (hunk 1 above)
Comment (mimugmail) [reply security/clamav/src/opnsense/mvc/app/models/OPNsense/ClamAV/Url.xml]: good idea! shouldn't this work? `<mask>/^https?:\/\/*$/i</mask>`

Comment (fabianfrz) [reply security/clamav/src/opnsense/mvc/app/models/OPNsense/ClamAV/Url.xml]: Add a dot before the asterisk. Then it means any charachter 0 to infinity times.

File: security/clamav/src/opnsense/mvc/app/views/OPNsense/ClamAV/general.volt (hunk 2 above)
Comment (fabianfrz) [reply security/clamav/src/opnsense/mvc/app/views/OPNsense/ClamAV/general.volt]: That may be a question of taste. If you want it as is, it's your plugin :)

File: security/clamav/src/opnsense/mvc/app/models/OPNsense/ClamAV/Url.xml (hunk 1 above)
Comment (mimugmail) [reply security/clamav/src/opnsense/mvc/app/models/OPNsense/ClamAV/Url.xml]: Thx, fixed

File: security/clamav/src/opnsense/mvc/app/views/OPNsense/ClamAV/general.volt (hunk 2 above)
Comment (mimugmail) [reply security/clamav/src/opnsense/mvc/app/views/OPNsense/ClamAV/general.volt]: I changed to `Signatures`

**Comment by mimugmail** [comment]
Also fixed a typo for JURLBLA signatures, should be ready now.

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**